robot-server/tests/conftest.py

```diff
@@ -92,6 +93,14 @@ def hardware() -> MagicMock:
     return MagicMock(spec=API)
 
 
+@pytest.fixture
+def versions() -> MagicMock:
+    m = MagicMock(spec=ComponentVersions)
+    m.api_version = "someTestApiVersion"
+    m.system_version = "someTestSystemVersion"
+    return m
```

Comment:
It looks like this could be a real `ComponentVersions` instead of a mock?

```suggestion
@pytest.fixture
def versions() -> ComponentVersions:
    return ComponentVersions(
        api_version="someTestApiVersion",
        system_version="someTestSystemVersion",
    )
```